app/views/followships/update.js.erb

```diff
@@ -0,0 +1,10 @@
+alert("here is")
+$('.btn').bind('ajax:success',function() {
+
+  
+  $(this).closest('.row').fadeOut()
```

Comment:
`row` is bootstrap class. Write you own one.And Use that.

Reply:
Done